bin/Makefile_ROCm

```diff
@@ -0,0 +1,78 @@
+# Makefile for Cray (XT5, XE/XK, XC90) with Cray Compiling Environment (CCE)
+# If PrgEnv-cray is not the default, use "module swap" to PrgEnv-cray
+
+FORTRAN_FIXEDFORM    = -ffixed
+FORTRAN_BOUND_CHECK  = #-R bcp #-- FIXME: bound checking causes issue with
+                               #          OpenMP. Reported to Cray.
+FORTRAN_DEBUG        = -g -O0 #-hdevelop -eF -hfp0 -K trap=fp
+FORTRAN_OPTIMIZE     = -O1
```

Comment:
Should be O2 I think

Reply:
If we change it to O2 we will run into segmentation fault while running RiemannProblem test program.

Comment from Reuben confirms the same
"> With just CPU OpenMP multithreading with ROCm-4.5.0 on Crusher, Basics RiemannProblem seems to run fine. However, with anything higher than -O1, I get segfault when it starting evolution."

So for time being lets keep it to -O1?